websites/V/Void Bots/presence.ts

```diff
@@ -0,0 +1,76 @@
+const presence = new Presence({ clientId: "765261270814949417" }), 
+  browsingStamp = Math.floor(Date.now() / 1000);
+let oldLang: string = null,
+  strings: any = {}
+
+function getMeta(metaName: string): string {
+  metaName = "PreMiD_"+metaName;
+  const metas = document.getElementsByTagName('meta');
+  for (let i = 0; i < metas.length; i++) { if (metas[i].getAttribute('name') === metaName) { return metas[i].getAttribute('content'); } }
+  return '';
+}
+
+function hasMeta(metaName: string): boolean {
+  metaName = "PreMiD_"+metaName;
+  const metas = document.getElementsByTagName('meta');
+  for (let i = 0; i < metas.length; i++) { if (metas[i].getAttribute('name') === metaName) { return true; } }
+  return false;
+}
+
+async function translateStrings(strings: any) {
+  return await presence.getStrings(strings, oldLang)
```

Comment:
Would this/does this work? as you never provide oldLang as an arg?

Reply:
It does work but I am going to remove that function because it's only used once and could easily be placed within the event.